app/helpers/ui_constants.rb

```diff
@@ -448,10 +448,10 @@ module UiConstants
   ASSIGN_TOS["ExtManagementSystem"] = {
     "enterprise"                 => N_("The Enterprise"),
     "ext_management_system"      => PostponedTranslation.new(N_("Selected %{tables}")) do
```

Comment:
The real change here should just be:

```ruby
N_("Selected Providers")
```

(same below) + fix the spec accordingly and leave `locale/en.yml` as it is.